README.md

```diff
@@ -9,7 +9,7 @@ Documentation
 -------------
 
 Mongoid has [extensive user documentation](https://docs.mongodb.com/mongoid/current/).
-[API documentation](http://api.mongodb.com/mongoid/current/) is also available.
+[API documentation](https://docs.mongodb.com/mongoid/current/api/) is also available.
```

Comment:
I think we're better off keeping it pointing to `current` to make sure there are no mistakes when the docs are released.

Reply:
I like that we have same concerns 😃 I was also thinking about this, and decided to keep `current` here because I am not sure that "master" documentation is actually up-to-date with the master branch. I would expect that master documentation is updated every time we merge to master. I believe this does not happen - the actual "master" documentation is `Generated on Thu Jul 15 18:01:56 2021`